source/compatibility.tex

```diff
@@ -1597,7 +1597,7 @@
 \grammarterm{pp-number} can contain \tcode{p} \grammarterm{sign} and
 \tcode{P} \grammarterm{sign}.
 \rationale
-Necessary to enable hexadecimal floating-point literals.
+Necessary to enable \grammarterm{hexadecimal-floating-point-literal}{s}.
```

Comment:
No braces around `s` please.

Reply:
Fixed. (We should probably fix that globally, later.)